lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
if this ends up expanding to a bunch of new smoke tests, are you thinking about moving them somewhere else/called somehow else instead of in the midst of all the (mostly) unit tests?

Reply:
called somehow else, yes definitely 👍 

moved somewhere else, no wasn't planning on it.